test/e2e/suite/conformance/certificates/acme/acme.go

```diff
@@ -153,6 +163,27 @@ func (a *acmeIssuerProvisioner) createHTTP01Issuer(f *framework.Framework) cmmet
 	}
 }
 
+func (a *acmeIssuerProvisioner) createHTTP01IssuerOnIstio(f *framework.Framework) cmmeta.ObjectReference {
+	a.ensureEABSecret(f, "")
+
+	By("Creating an ACME HTTP01 Issuer")
+	issuer := &cmapi.Issuer{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "acme-issuer-http01-",
+		},
+		Spec: a.createHTTP01IssuerOnIstioSpec(f.Config.Addons.ACMEServer.URL),
+	}
+
+	issuer, err := f.CertManagerClientSet.CertmanagerV1().Issuers(f.Namespace.Name).Create(context.TODO(), issuer, metav1.CreateOptions{})
+	Expect(err).NotTo(HaveOccurred(), "failed to create acme HTTP01 issuer")
```

Comment:
`	Expect(err).NotTo(HaveOccurred(), "failed to create acme HTTP01 Istio issuer")`